skbio/embedding/_embedding.py

```diff
@@ -0,0 +1,39 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import numpy as np
+
+from skbio.sequence import Sequence
+from skbio._base import SkbioObject
+from skbio.metadata._mixin import MetadataMixin
+
+
+class Embedding(SkbioObject):
```

Comment:
I think the current structure (embedding is a 1st-level module) is great. Just wanted to ask if there are non-sequence embeddings envisioned in our plan.

Reply:
Yes, once we have pLMs in place, we will want to think about molecular and mass-spectrometry embeddings / vectors

Reply:
Cool! Two more questions: 1) Will ordinations be considered as embeddings? 2) We need to think about whether we do: sequence -> sequence embedding, or embedding -> sequence embedding. Since embedding can host multiple heterogerous biological objects, I wonder if it makes more sense to separate them under sequence, structure, spectrum, etc. But I also see the point of other way round. Hard question...

Reply:
As discussed previously, let's do embedding -> sequence embedding
The reason being is that there are some attributes in sequence that we don't want to expose in sequence embedding (i.e. positional indexing for sequence objects doesn't make much sense for sequence embeddings, since the embeddings are generated using the entire sequence, the context is important).